lib/screens/v2/location_context.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Screens.LocationContext do
+  @moduledoc false
+
+  alias Screens.Routes.Route
+  alias Screens.RouteType
+  alias Screens.Stops.Stop
+
+  defstruct home_stop: "",
```

Comment:
If a struct is created without one of these attributes and the default is used, will all logic still work as expected? If not, may be helpful to move some of the attributes into a `@enforce_keys` list.

Reply:
Hmm, I see. I suppose I might as well enforce all keys for now, and we can un-enforce any that pose a problem in the future. 

Using the default `home_stop` of `""` is fine for GL e-ink, but it breaks the Elevator widget. Using the default `[]` for `stop_sequences` doesn't break any screen, but it does mess up how the alert displays on DUPs. Does that mean we need to enforce `home_stop`, but don't necessarily need to enforce `stop_sequences`, or must we enforce both?

Reply:
I can see an argument for either way, but I would lean towards just enforcing attributes that need to be there for the screen to work. If the default throws off the visuals, the dev working with the struct should be able to catch that on their own.